### PR TITLE
New package: tarpaulin-0.9.0

### PR DESCRIPTION
**Pull request by jcgruenhage**

### CODE REVIEW

File: srcpkgs/tarpaulin/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'tarpaulin'
+pkgname=tarpaulin
+version=0.9.0
+revision=1
+archs="x86_64"
```
Comment (Johnnynator) [inline srcpkgs/tarpaulin/template]: Can you clarify why it is only for x86_64(-glibc)? Shouldn't it work at least traditionally on x86_64-musl?

Comment (jcgruenhage) [reply srcpkgs/tarpaulin/template]: How would I enable this? Setting archs to "x86_64 x86_64-musl"?

Comment (jcgruenhage) [reply srcpkgs/tarpaulin/template]: (Yes, it should work on musl)

Comment (Johnnynator) [reply srcpkgs/tarpaulin/template]: You can use wildcards `x86_64*`

Comment (jcgruenhage) [reply srcpkgs/tarpaulin/template]: done :heavy_check_mark:

**Pull request merged**